doc/source/tutorials/machine_learning/mnist_classifier.ipynb

```diff
@@ -631,7 +649,7 @@
    "hash": "197454157c080c19c876ac9bd28ff3f683250ebc7e59376e497fe430017b1105"
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.8.13 ('cova-qa-1')",
```

Comment:
what is `cova-qa-1`?

Reply:
This was an old change... I've fixed this by syncing this with the version on develop